src/TritonRoute/src/dr/FlexDR_maze.cpp

```diff
@@ -1898,9 +1898,9 @@ void FlexDRWorker::route_queue_main(queue<RouteQueueEntry>& rerouteQueue)
         // TODO: output maze area
         cout << "Fatal error: Maze Route cannot find path ("
              << net->getFrNet()->getName() << ") in "
-             << "(" << routeBox.left() / 2000.0 << ", "
```

Comment:
Probably a hardcoded conversion from DBU to um.

Reply:
Would it be best then to add units to this log message either in terms of DBU or performing an actual unit conversion to some known unit like µm?

Reply:
Probably. I spotted this already but haven't gotten around to it yet. There's a `dbuToMicrons()` function.

Reply:
It also need to be converted to and error() not cout.

Reply:
The probable reason is that this was considering ISPD benchmarks, which have 2000 dbu per micron. I really dont like changing the output to microns, since it is one more thing to do (translate back to dbu) for the one who is going to debug (all code is in dbu). I will convert to an error from the logger.

Reply:
One possibility is to simply output both um and DBU.

Reply:
Yes, but for coherence, this should be done in other outputs, and this goes beyond the purpose of this fix. I just removed the /2000 in this fix because I came across it and this is misleading, and already made me waste time tracking the wrong info. 

Another problem is that this would make the code even more polluted with double prints.

Reply:
I think just adding `DBU` after the number would be helpful even if we don't want to change other areas yet. While I understand the desire for consistency slow improvement is better than no improvement in this case; in my opinion.